Makefile

```diff
@@ -509,4 +509,7 @@ vsphere-management-and-workload-cluster-e2e-test:
 unmanaged-cluster-e2e-test:
 	cd cli/cmd/plugin/unmanaged-cluster/test/e2e && BUILD_VERSION=$(BUILD_VERSION) go test -test.v -timeout 180m
 
+diagnostic-e2e-test:
+	cd cli/cmd/plugin/diagnostics/e2e-test && BUILD_VERSION=$(BUILD_VERSION) ROOT_DIR=$(ROOT_DIR) go test -timeout 180m
```

Comment:
See note below, but also remove passing through `$ROOT_DIR` here and just have the plugin's Makefile look it up if needed.